rep-2003.rst

```diff
@@ -0,0 +1,76 @@
+REP: 2003
+Title:  Sensor Data and Map QoS Settings
+Author: Steve Macenski <stevenmacenski at gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 08-Nov-2019
+ROS-Version: Dashing
+Post-History: 08-Nov-2019
+
+Abstract
+========
+
+This REP covers the standards surrounding Quality of Service (QoS) settings for common needs in ROS2.
+This includes sensor drivers and map representations to standardize interfaces of not only message types outlined by REPs such as REP 118, 138, and 145, but also QoS to ensure ROS2 interoperability.
+
+Specification
+=============
+
+Map Quality of Service
```

Comment:
It may be worth describing what transient-local is for people who aren't familiar in this section.

Reply:
Good point

Reply:
Well, actually @mjcarroll do you think that a definition is really required on a standard? There's  documentation on ROS2 about what the settings mean in particular, the goal of this is to define the standard set of practices. I don't know that this is the right place to be teaching them about what are the QoS "basics".

Reply:
I would be okay with a simple analogy or description on first introduction, something like:  
> reliable transient-local (which saves and re-delivers the last valid message to subscribers added later)

I only say that because googling "transient-local" isn't particularly informative and is a bit opaque for folks that aren't versed in the DDS terminology.

Reply:
> This is to allow for reliable transmission of map data and provide the map information to any new map clients without delay.


Doesn't that get the same message across in a more approachable way?

Reply:
Ah yes, it does. Sorry for the noise.

Reply:
No worries :+1: